core-java-modules/core-java-string-operations-2/src/test/java/com/baeldung/encode/StringEncodeUnitTest.java

```diff
@@ -0,0 +1,42 @@
+package com.baeldung.encode;
+
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.nio.charset.StandardCharsets;
+
+import org.junit.Test;
+
+public class StringEncodeUnitTest {
+
+    private String unencodedString = new String("Entwickeln Sie mit Vergnügen".getBytes(), StandardCharsets.US_ASCII);
```

Comment:
Wouldn't it be more accurate to call this variable asciiEncodedString?

Reply:
Sure.